nuxt.config.ts

```diff
@@ -51,8 +51,8 @@ export default defineNuxtConfig({
   },
 
   runtimeConfig: {
-    public: {
-      GQL_HOST: ''
```

Comment:
i te?

Reply:
Tak samo jak wczesniej. nie mam pojecia skad to sie wzielo